docs/content/configuration/global-configuration/command-line-arguments.md

```diff
@@ -164,12 +164,8 @@ Sets the URI of health status location in the default server. Requires [-health-
 
 ### -ingress-class `<string>`
 
-A class of the Ingress Controller.
-
-A corresponding IngressClass resource with the name equal to the class must be deployed. Otherwise, the Ingress Controller will fail to start.
-The Ingress Controller only processes resources that belong to its class - i.e. have the `ingressClassName` field resource equal to the class.
-
-The Ingress Controller processes all the resources that do not have the `ingressClassName` field.
+This argument refers to the name of resource `kind: IngressClass`. An IngressClass resource with the name equal to the class must be deployed. Otherwise, the Ingress Controller will fail to start.
+The Ingress Controller will process Ingress resources that belong to its class i.e. have the `ingressClassName` field equal to the value of `-ingress-class` and skip the ones without it, but processes all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the `ingressClassName` field.
```

Comment:
```suggestion
NGINX Ingress Controller will process ingress resources that belong to its class, that is, those that have the `ingressClassName` field equal to the value of `-ingress-class` and skip the ones without it. It will also process all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the `ingressClassName` field.
```

Avoid Latinisms like i.e., e.g.